gdk-pixbuf/src/pixbuf.rs

```diff
@@ -306,6 +306,16 @@ impl Pixbuf {
         }))
     }
 
+    // rust-doc-stripper-ignore-next
+    /// Returns a mutable slice to the pixbuf's pixel data.
+    ///
+    /// This function will cause an implicit copy if the pixbuf was created from read-only data.
+    ///
+    /// Please see the section on [image data](#image-data) for information about how the pixel
+    /// data is stored in memory.
+    ///
+    /// # Safety
+    /// No other reference to this pixbuf must exist when this method is called.
```

Comment:
Well it's fine to have more refernces to the pixbuf. But there must be no other reference to its data, and none of the modifying/reading pixbuf operations must be called (as they also access the pixels)

Reply:
Do you think the new safety section is better?

Reply:
Yes, that's good now :)